File: Folder Java/Kode/Aula.java
// Aula.java
class Aula extends TempatUsaha {
    private double luas;
    private String kapasitas;
    private String fasilitas;
    private double hargaSewa;
    private String tipe;

    public Aula() {}
    
    public Aula(String namaTempat, String noIzin, String kategoriUsaha, String lokasi, String deskripsi,
                double luas, String kapasitas, String fasilitas, double hargaSewa, String tipe) {
        super(namaTempat, noIzin, kategoriUsaha, lokasi, deskripsi);
        this.luas = luas;
        this.kapasitas = kapasitas;
        this.fasilitas = fasilitas;
        this.hargaSewa = hargaSewa;
        this.tipe = tipe;
    }
}

File: Folder Python/Kode/Aula.py
# Aula.py
from TempatUsaha import TempatUsaha

class Aula(TempatUsaha):
    def __init__(self, nama_tempat="", no_izin="", kategori_usaha="", lokasi="", deskripsi="",
                 luas=0.0, kapasitas="", fasilitas="", harga_sewa=0.0, tipe=""):
        super().__init__(nama_tempat, no_izin, kategori_usaha, lokasi, deskripsi)
        self.__luas = luas
        self.__kapasitas = kapasitas
        self.__fasilitas = fasilitas
        self.__harga_sewa = harga_sewa
        self.__tipe = tipe